implementations/rust/ockam/ockam_api/src/authority_node/authority.rs

```diff
@@ -334,7 +345,331 @@ impl Authority {
+            database_configuration: DatabaseConfiguration::postgres()?.unwrap(),
+            project_identifier: "123456".to_string(),
+            tcp_listener_address: InternetAddress::new(&format!("127.0.0.1:{}", port)).unwrap(),
+            secure_channel_listener_name: None,
+            authenticator_name: None,
+            trusted_identities: PreTrustedIdentities::new(trusted_identities),
+            no_direct_authentication: false,
+            no_token_enrollment: false,
+            okta: None,
+            account_authority: None,
+            enforce_admin_checks: false,
+            disable_trust_context_id: false,
+        })
+    }
+
+    /// Make a client to access the services of an Authority
+    async fn make_authority_node_client(
+        ctx: &Context,
+        secure_channels: Arc<SecureChannels>,
+        authority_identifier: &Identifier,
+        authority_route: &MultiAddr,
+        caller: &Identifier,
+    ) -> Result<AuthorityNodeClient> {
+        let client = NodeManager::authority_node_client(
+            &TcpTransport::create(ctx)?,
+            secure_channels,
+            authority_identifier,
+            authority_route,
+            caller,
+            None,
+        )
+        .await?;
+        Ok(client
+            .with_secure_channel_timeout(&Duration::from_secs(1))
+            .with_request_timeout(&Duration::from_secs(1)))
+    }
+
+    /// Create and start an authority node, with:
+    ///  - A specific TCP listener port
+    ///  - A specific node name
+    ///  - An identifier for an enroller
+    async fn start_authority_node(
+        db: SqlxDatabase,
+        ctx: &Context,
+        port: u16,
+        node_name: &str,
+        trusted: &[Identifier],
+    ) -> Result<Authority> {
+        let identities = identities::create(db.clone(), node_name);
+        let authority = identities.identities_creation().create_identity().await?;
+
+        let configuration = create_configuration(&authority, port, trusted)?;
+        let authority = Authority::create(&configuration, Some(db.clone())).await?;
+        authority_node::start_node(ctx, &configuration, authority.clone()).await?;
+        Ok(authority)
+    }
+
+    /// Add a member
+    async fn add_member(
+        ctx: &Context,
+        client: &AuthorityNodeClient,
+        member: &Identifier,
+    ) -> Result<()> {
+        if let Err(e) = client
+            .add_member(ctx, member.clone(), BTreeMap::new())
+            .await
+        {
+            panic!("{e:?}");
+        };
+        Ok(())
+    }
+
+    /// Issue a credential for a given member
+    async fn issue_credential(
+        ctx: &Context,
+        client: &AuthorityNodeClient,
+        member: &Identifier,
+    ) -> Result<()> {
+        if let Err(e) = client
+            .clone()
+            .with_client_identifier(member)
+            .issue_credential(ctx)
+            .await
+        {
+            panic!("{e:?}");
+        };
+        Ok(())
```

Comment:
Wouldn't be the same if the function doesn't return a Result and instead you just unwrap the function call (same for the `add_member` function)?

`client.clone().with_client_identifier(member).issue_credential(ctx).await.unwrap()`

Reply:
Right, going to do this.